tests/login.test.js

```diff
@@ -6,7 +6,7 @@ import { expect } from 'chai';
 describe('OpenBank', () => {
 
     
-    it('Should access the home page when attempting to login with valid credentials', async () => {
+    it.skip('Should access the home page when attempting to login with valid credentials', async () => {
```

Comment:
Is there a reason for skipping all login tests?

Reply:
Login TCs were skipped just for this sprint, as I mentioned in the description of this pull request...
Is there another way to run the logout tests only?

Reply:
I think the login tests were shown last sprint so there's no need to show them again...

Reply:
The repository is tipically accumulative (you are building a general regression suite) and execution patterns are typically managed at a different level (for example, using a CLI command with a specific flag signaling what suite you'd like to execute). This usage of **skip** and **only** is more typically used for debugging purposes, but removed when the feature is finished.

Reply:
Done!